packages/form-js-editor/src/render/components/palette/Palette.js

```diff
@@ -1,57 +1,57 @@
-import { Fragment } from 'preact';
-
-import { iconsByType } from './icons';
-
-const types = [
-  {
-    label: 'Text Field',
-    type: 'textfield'
-  },
-  {
-    label: 'Number',
-    type: 'number'
-  },
-  {
-    label: 'Checkbox',
-    type: 'checkbox'
-  },
-  {
-    label: 'Radio',
-    type: 'radio'
-  },
-  {
-    label: 'Select',
-    type: 'select'
-  },
-  {
-    label: 'Text',
-    type: 'text'
-  },
-  {
-    label: 'Button',
-    type: 'button'
-  }
-];
-
-
-export default function Palette(props) {
-  return <Fragment>
-    <div class="fjs-palette-header">FORM ELEMENTS LIBRARY</div>
-    <div class="fjs-palette fjs-drag-container fjs-no-drop">
-      {
-        types.map(({ label, type }) => {
-          const Icon = iconsByType[ type ];
-
-          return (
-            <div class="fjs-palette-field fjs-drag-copy fjs-no-drop" data-field-type={ type }>
-              {
-                Icon ? <Icon class="fjs-palette-field-icon" width="36" height="36" viewBox="0 0 54 54" /> : null
-              }
-              <span>{ label }</span>
-            </div>
-          );
-        })
-      }
-    </div>
-  </Fragment>;
+import { Fragment } from 'preact';
```

Comment:
What did we change here?

Reply:
This file was moved and renamed without any changes.